### DISCUSSION TITLE
Doses de frontend

### DISCUSSION THREAD

**Discussion by oguilhermelessa** [frontendbr/forum#461] [Dicas]
Fala galera,
boa noite.

Então, eu criei um grupo no Whatsapp chamado **Doses de Frontend**. O intuito desse grupo é criar um networking entre o povo focado em frontend. A ideia é que todos ensinem, por isso o nome de **Doses de Frontend**. Cada um coloca um pouquinho do que sabe, experiências, e etc...

Não vou colocar aqui agora, o link grupo para acesso, pois não sei se é permitido. Se não for por favor, administradores, me perdoem e apaguem o post, mas se não, e se quiserem se juntar, tamo junto!

Depois da permissão eu começo esse start!

Abraço,
fiquem na paz.

---------

Atualização (17/fev)

Segue o link do grupo galera, aprovado pelos administradores.

https://chat.whatsapp.com/7c2uNrlJdzQ6xFzalKnYw5

**Comment by fdaciuk** [comment]
Não sei se é legal pedir informações pessoais por aqui, pois tudo fica público, mas você poderia chamar quem quisesse para te enviar o número no slack do Frontend Brasil. Assim as pessoas que quiserem entrar no grupo podem te enviar o número no privado =)

O que acham @frontendbr/admins ?

**Comment by eduardojmatos** [comment]
@guilessa @fdaciuk acho uma boa ideia! Mas colocar número de tel aqui seria ruim mesmo. A ideia do @fdaciuk é bem boa!

**Comment by oguilhermelessa** [comment]
Não, na verdade o whatsapp agora cria link para convidados. Eu que colocaria o link do grupo aqui, a pessoa entra pelo Whatsapp Web e automaticamente já faz parte do grupo.

**Comment by felipefialho** [comment]
Pode colocar então :)

**Comment by ranosi** [comment]
No aguardo :)

2017-02-17 12:22 GMT-02:00 Felipe Fialho <notifications@github.com>:

> Pode colocar então :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/461#issuecomment-280662488>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AV1hh1uY3ZqveGOCdTNY-L6vj2jj8Lcpks5rda06gaJpZM4MCdHs>
> .
>



-- 
Grande abraço,



[image: --]

Rafael Nogueira
[image: https://]about.me/rafaelnogueira
<https://about.me/rafaelnogueira?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>

**Comment by oguilhermelessa** [comment]
Segue o link do grupo galera!

https://chat.whatsapp.com/7c2uNrlJdzQ6xFzalKnYw5
